networks/devnet/main/05tomoscan.yml

```
@@ -19,6 +19,8 @@ services:
       MONGODB_URI: mongodb://tomoscan_db:27017/explorer
       REDIS_HOST: tomoscan_redis
       CLIENT_URL: https://scan.devnet.tomochain.com/
+      TOMOMATER_URL: http://tomomaster:3001
```

```suggestion
      TOMOMASTER_URL: http://tomomaster:3001
```

---

Or even:
```suggestion
      TOMOMASTER_API_URL: http://tomomaster:3001
```
To be consistent with https://github.com/tomochain/infrastructure/pull/92#discussion_r233160176

---

@pqv199x , update the code-based, TOMOMASTER_URL -> TOMOMASTER_API_URL

---

https://github.com/tomochain/tomoscan/pull/487